cts-java/src/test/java/org/ga4gh/cts/api/variants/CallsetsSearchResponseCheckIT.java

```diff
@@ -56,7 +56,41 @@ public void searchForExpectedCallSets() throws AvroRemoteException {
             assertThat(csResp.getCallSets()).isNotEmpty();
         }
     }
+    /**
+     * Make sure VariantSet records exist for each CallSet.
+     *
+     * @throws AvroRemoteException if there's a communication problem or server exception ({@link GAException})
+     */
+    @Test
+    public void searchForExpectedVariantSetsByCallSets() throws AvroRemoteException {
```

Comment:
One minor quibble: The name of this test makes it sound like you're searching for variant sets. Instead, you're using a variant set as the query to search for call sets, then verifying that these call sets contain that variant set. Wouldn't something like `testSearchCallSetsByVariantSet` be more appropriate?